test/app-luatest/override_misc_test.lua

```diff
@@ -52,3 +52,21 @@ g.test_initializing = function(g)
     local exp = expected_output('socket')
     t.assert_equals(res, exp)
 end
+
+g.test_no_package_loaded = function(g)
```

Comment:
`g` is unused, see luacheck warnings.

Reply:
Sorry for this mess. I was in a hurry a bit, but this doesn't excuses me. Fixed.

Reply:
It's OK, we are not robots (at least most of us):).